### ISSUE TITLE
Build failing on my fork

### ISSUE THREAD

**Issue by HaiziIzzudin** [j-hc/revanced-magisk-module#92]
Hi, recently my Action build fails since the repo upstream update on my fork. Below is logs:

![image](https://user-images.githubusercontent.com/79714350/200352880-d46745bc-f485-4e1d-abcf-26808bbb5509.png)

Also my fork contains modified build.conf file. Below is the code:

```
YOUTUBE_EXCLUDED_PATCHES=""
YOUTUBE_INCLUDED_PATCHES="video-ads minimized-playback old-quality-layout remember-video-quality"
MUSIC_EXCLUDED_PATCHES=""
MUSIC_INCLUDED_PATCHES=""

# MODE: build-mode/version
# options: [module|apk|both|none]/[auto|latest|custom (like 17.36.42)]
# examples: module/auto, both/17.40.41, apk/latest, module/17.36.42, apk/auto, none/none
# 'auto' = selects according to the latest revanced-patches supported version
# 'latest' = always latest
YOUTUBE_MODE=module/auto
MUSIC_ARM64_V8A_MODE=none/auto
MUSIC_ARM_V7A_MODE=none/auto
TWITTER_MODE=none/latest
REDDIT_MODE=none/latest
TIKTOK_MODE=none/latest

SPOTIFY_MODE=none/latest # patches for spotify are quite useless tbh
WARN_WETTER_MODE=none/latest

MOUNT_DELAY=5             # some low-end phones cannot mount apks on boot without a delay. increase to something like 20 if needed.
COMPRESSION_LEVEL=9       # compression level for module zips. between 1 and 9
ENABLE_MAGISK_UPDATE=false # set this to false if you do not want to receive updates from magisk app
BUILD_MINDETACH_MODULE=false

UPDATE_PREBUILTS=true # dont touch this
```

Any problem with my code/configs? 😲

**Comment by j-hc** [comment]
should be fixed now with 9cbffa46671fe4ab59f1b3bc292b2e0f00c0bf20

**Comment by HaiziIzzudin** [comment]
![image](https://user-images.githubusercontent.com/79714350/200362597-b9edc3af-9ddd-488b-b786-81240d51221e.png)

New problem rising, but this time the build successfully get pushed to Releases. Mybe this only just information about the build, that it failed?

**Comment by j-hc** [comment]
fixed now 9068ffd
